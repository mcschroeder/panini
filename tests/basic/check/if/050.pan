
f50 : {v:int|false}
f50 = if true then 1 else 2

f51 : {v:int|false}
f51 = if false then 1 else 2

f52 : {v:int|false}
f52 = let b = true in if b then 1 else 2

f53 : {v:int|false}
f53 = let b = false in if b then 1 else 2
