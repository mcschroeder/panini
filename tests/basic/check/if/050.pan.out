tests/basic/check/if/050.pan:3:1-4: error: cannot solve constraints of f50
tests/basic/check/if/050.pan:6:1-4: error: cannot solve constraints of f51
tests/basic/check/if/050.pan:9:1-4: error: cannot solve constraints of f52
tests/basic/check/if/050.pan:12:1-4: error: cannot solve constraints of f53